addressbook/interface/exchangeAutoCompleteResult/mivExchangeAutoCompleteResult.js

```diff
@@ -116,20 +112,15 @@ mivExchangeAutoCompleteResult.prototype = {
      */
     //readonly attribute long defaultIndex;
     get defaultIndex() {
-        //dump("defaultIndex\n");
-        if (this._cards.length == 0) {
-            return -1;
-        }
-
-        return 0;
+        // by default index is always 0 if the _idcards contains at least one element
+        return this._idcards.length > 0 ? 0 : -1;
```

Comment:
A bit pedantic, but the cleaner version also works just fine:
```
return this._idcards.length ? 0 : -1;
```

Reply:
Mmhh, I made a test yesterday with the Javascript console, before writing this.

I've tried again this evening and it works as you except, I certainly made a mistake with my test yesterday, I'll change it again.